pages/index.page.tsx

```diff
@@ -206,7 +207,7 @@ const Home = (props: any) => {
   const [airbnb_logo, setAirbnb_logo] = useState('');
   const [postman_logo, setPostman_logo] = useState('');
   const [itflashcards_logo, setItflashcards_logo] = useState('');
-  const [route4me_logo, setRoute4me_logo] = useState('');
```

Comment:
Why do we no longer need this?

Reply:
sure , i will resolve these issues and will raise pr soon

Reply:
You don't have to close this PR, you can continue adding commits to this.